bashttpd

```diff
@@ -274,9 +285,13 @@ while read -r line; do
 
    # If we've reached the end of the headers, break.
    [ -z "$line" ] && break
+   
+   [ "$REQUEST_METHOD" = "POST" ] && [[ $line == Content-Length:* ]] && POST_CONTENT_LENGTH=${line##* }
 
    REQUEST_HEADERS+=("$line")
 done
 
+[ "$REQUEST_METHOD" = "POST" ] && read -N $POST_CONTENT_LENGTH POST_PARAMETERS
```

Comment:
You might want to read here with and `-r` to not allow backslashes in the request body content to escape any characters

Reply:
Thank you for your contribute.

Bear in mind this project is not recommended for 'production environments' at all. This is just a proof of concept, good for prototyping, and at best, something you could use on a safe environment such as a small lab network, intranet, or some IPv4 island of your own.

That being said, i consider the 'code injection' as a pretty neat feature, not a bug. Thanks for helping pointing that out to others.